src/psyclone/dynamo0p3.py

```diff
@@ -8572,7 +8572,9 @@ class creates the PSyIR schedule on first invocation which is
 
         if len(routines) == 1:
             sched = routines[0]
-            self.validate_kernel_code_args(sched.symbol_table)
+            # We don't validate the arguments yet because the validation has
+            # many false negatives.
```

Comment:
Please could you add a TODO with a reference to the issue(s) that will fix this. e.g. #928.

Reply:
Done